signing_service/src/signing_service.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import logging.config
+import os
+import socket
+
+
+logger = logging.getLogger()
+crash_logger = logging.getLogger('crash')
+
+
+def main(concent_cluster_address):
+    address, port = concent_cluster_address.split(':')
+    port = int(port)
```

Comment:
I assumed we would not have to parse the address. I don't want to go into various special cases like missing port, etc.

Let's switch to separate parameters for address and port. I'll update the issue.

Reply:
Can it also be done in #634 ?

Reply:
It's a simple change. Let's do it here.

I'd prefer not to add unnecessary noise to #634.

Reply:
:+1 done